docs/use-cases/support.mdx

```diff
@@ -12,6 +13,10 @@ Customer support is an integral part of any organization, serving as the bridge
 XMTP‍ presents a new opportunity to revolutionize customer support by providing encrypted communication channels. By leveraging the features of XMTP, customer support can be taken to new heights, balancing both user convenience and information security.
 
 <div class="chain-list white">
+  <a href="https://www.withblaze.app/" class="chain-item">
```

Comment:
nit: do you want to add a `target="_blank"` here to open this in a separate tab?

Reply:
Love this - thank you and wonder of wonders - Docusaurus opens external links in a new tab by default! <3